packages/components/Footer.tsx

```diff
@@ -1,64 +1,51 @@
 "use client";
 
-import { Stack, Text, Image, Grid } from "@chakra-ui/react";
-import { css } from "@emotion/react";
-
-import { useI18n } from "@packages/features/i18n-context";
-import { PIX_KEY, images } from "@packages/config/site";
 import { Logo } from "@packages/components/icons";
+import { PIX_KEY, images } from "@packages/config/site";
+import { useI18n } from "@packages/features/i18n-context";
 
 import Section from "./Section";
 import SocialIconLinks from "./SocialIconLinks";
 
 export default function Footer() {
   return (
     <Section className="bg-neutral-950 px-6 py-6 items-center">
-      <Grid gridTemplateColumns={{ base: "1fr", sm: "repeat(3, 1fr)" }} color="gray.50" css={centerGridCells}>
+      <div className="grid grid-cols-3 text-gray-50">
```

Comment:
suggestion: take a look at tailwind css primitives
- https://tailwindcss.com/docs/responsive-design

```suggestion
      <div className="grid grid-cols-1 sm:grid-cols-3 text-gray-50">
```